stylesheets/utilities.css

```diff
@@ -0,0 +1,67 @@
+/* padding */
+
+/* 1rem */
+.p-b-1{
+    padding-bottom: 1rem;
+}
+
+.p-l-1{
+    padding-bottom: 1rem;
+}
+
+.p-r-1{
+    padding-bottom: 1rem;
+}
```

Comment:
you have given the same style for different class

Reply:
fixed the mistake.